tests/negative/issue1700/SelfApplication.juvix

```diff
@@ -1,5 +1,5 @@
 -- self-application
-module test019;
```

Comment:
I would also leave this in Compilation/positive/test019.juvix, because we want to later check compilation of this down to WASM.

Reply:
I restored the test.